### DISCUSSION TITLE
Click/touch inputs not registering

### DISCUSSION THREAD

**Discussion by staticsh33p** [jeffreylanters/react-unity-webgl#392] [Help Wanted!] [answered]
Hey there!
New to the React-Unity space, hello to all!
I got my project to load into UnityContext object and display correctly, but it looks like all of the buttons aren't handling any click or touch input...anyone encounter this? Any ideas?
Thanks!!

**Comment by jeffreylanters** [comment, marked as answer]
Hi, welcome! Could you use the element inspector to validate no element or CSS rule is blocking you from virtually clicking or interacting with the canvas? Or maybe there's an error in the console which prevents you from interacting.

**Reply by staticsh33p** [reply]
Found it! (In App.css).
Thanks! I didn't know the pointer-events was "none" by default.
Much appreciated!
